communication/calendar-guidelines.md

```diff
@@ -95,6 +100,54 @@ will prevent you from getting updates like a canceled meeting. Join the main
 
 Accept the invite from the sender and you'll have the updates.
 
+### Calendar Event Template
+
+In case you are creating a calendar event and wondering what to include in it, here is a recommended template.
+
+#### Title
+
+```
+[SIG-<name>] <meeting title>
+or
+[WG-<name>] <meeting title>
+or
+[User Group-<name>]
+```
+#### Description
+
+```
+<One line intro about the SIG/ WG. Can include github links>
+
+Time: <Weekly/ biweekly/... or is the meeting for a fixed time and till when?>
+
+Meeting notes & agenda: <Hyperlink to the meeting notes doc>
+
+Zoom link: <Zoom invite link. Preferably both app and web client links>
```

Comment:
```suggestion
Zoom link: <Zoom invite link. Preferably include app client link, web client link, and dial in information>
```

Reply:
@rajula96reddy added that last nit of an idea. Lmk if you agree - either way it's ready to go once we wrap this up.

Reply:
@mbbroberg Yeah about this. I am cool with adding dial in information as well. But the dial in information varies from country to country. This might be confusing for the person creating the event. We can still add it though considering the user will add a default country. What do you say?

Reply:
Personally, I think we can skip the dial information and link out to their "find local dial in number page" - https://zoom.us/u/bacaZSMhA